README.md

```diff
@@ -1,4 +1,262 @@
+  - Before actually entering FCP, *all* members of the working group must sign off;
+  this is often the point at which many working group members first review the RFC
+  in full depth.
+- The FCP lasts ten calendar days, so that it is open for at least 5 business
+  days. It is also advertised widely,
+  e.g. in the [Weekly IPFS All Hands Call](https://github.com/ipfs/pm/#weekly-all-hands). This way all
+  stakeholders have a chance to lodge any final objections before a decision
+  is reached.
+- In most cases, the FCP period is quiet, and the RFC is either merged or
+  closed. However, sometimes substantial new arguments or ideas are raised,
+  the FCP is canceled, and the RFC goes back into development mode.
+
+### Status: Accepted    
+
+Once an RFC becomes "Accepted" then authors may implement it and submit the
+feature as a pull request to the corresponding repo. Being "Accepted" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "Accepted"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether an IPFS developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+### Modifications to Accepted RFCs
+
+Modifications to "Accepted" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the working group to decide.
+
+_If_ you make changes to an RFC after it's been accepted, add a note in the header of the RFC with a very short summary of the change and links to any relevant discussions. This way the change is very explicit and hard to miss.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the working group may schedule meetings with the
+author and/or relevant stakeholders to discuss the issues in greater detail,
+and in some cases the topic may be discussed at a working group meeting. In either
+case a summary from the meeting will be posted back to the RFC pull request.
```

Comment:
> from the meeting will be posted back to the RFC pull request

Maybe should be changed to:

> from the meeting **must** be posted back to the RFC pull request

Reply:
Just an idea: to remove ambiguity and make interpretation easier for non-native speakers we may consider prefacing the document with this disclaimer:

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and  "OPTIONAL" in this document are to be interpreted as described in  [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)

Reply:
switching to "must". @lidel I like your suggestion but not sure where in the document to put it. Do you have an idea of which section or which line I should add it to?

Reply:
got it. I added a paragraph in that section. switched up the language a bit.